### ISSUE TITLE
Manual page for lychee

### ISSUE THREAD

**Issue by thomas-zahner** [lycheeverse/lychee#1505]
As a user of lychee I'd like to read the manual with `man lychee` as is possible with other programs.

Personally, I've never written a man page before, but I can give it a shot when I find the time. Most content from the README can probably be reused. I guess the sections and structure would be the most difficult part to get right.

@mre do you have any experience with writing man pages?

**Comment by mre** [comment]
Good idea. I don't have any experience with man pages yet, but the README and the docs are a good start. 

As for tooling, here's a survey: https://stackoverflow.com/a/78226570

**Comment by mre** [comment]
@orhun mentioned two nice tools: https://github.com/lycheeverse/lychee/issues/1564#issue-2644869766
